### DISCUSSION TITLE
Update loaded data without pulling new objects in

### DISCUSSION THREAD

**Discussion by danvernon** [TanStack/query#2885] [Q&A]
I have a simple activity feed that has reactions + comments in React Native. I recently switched to React Query and have everything set up as it was working previously with redux.

The one thing I am struggling with is, I want to be able to refresh the current posts in the feed on a timer, like polling I suppose without bringing new posts in. 

I have right now a button that shows at the top when the new data has more objects than the old data, so we know someone has created a new post and I want to keep it that way. As when if I just enabled polling/refetch every 30 seconds if the new data comes in it messes the index of where the user is viewing if they are scrolling.

So thats why I need to keep the functionality of the refetch happening on user interaction, but somehow update the existing data with new updates.

**Comment by TkDodo** [comment]
sorry, I don't quite follow what the problem is. I think a codesandbox reproduction would be really helpful.
